### PR TITLE
Tiny Vale tweaks for Datasource and Logging guide

### PR DESCRIPTION
**Pull request by MichalMaler**
These are some tiny cosmetics that support the great effort of @michelle-purcell who incorporates all other writer's feedback on Vale behavior and its follow-up improvisation.

### CODE REVIEW

File: docs/src/main/asciidoc/datasource.adoc [hunk 1]
```diff
@@ -32,7 +32,7 @@ Additionally, refer to the Quarkus xref:hibernate-orm.adoc[Hibernate ORM] guide
 
 == Get started with configuring `datasources` in Quarkus
 
-For rapid configuration of datasources, this section offers a brief overview and code samples for testing and utilization, suitable for users familiar with the fundamentals.
+To quickly set up datasources, this section provides a concise overview and code samples for testing and usage, suitable for users with a basic understanding of the fundamentals.
```
Comment (jmartisk) [inline docs/src/main/asciidoc/datasource.adoc:35]: The note above says 

> In text, use "data source"

Would be good to adhere to it :)

**Review by jmartisk — CHANGES_REQUESTED** [review]

File: docs/src/main/asciidoc/datasource.adoc (hunk 1 above)
Comment (MichalMaler) [reply docs/src/main/asciidoc/datasource.adoc:35]: Patching :)

[the remark this answers is not in the document]
Comment (MichalMaler) [reply docs/src/main/asciidoc/datasource.adoc:35]: OK.

**Comment by MichalMaler** [comment]
THX @Ladicek n @jmartisk .
Applied your suggestions.
Cheers!

**Review by yrodiere — APPROVED** [review]

**Review by jmartisk — APPROVED** [review]

**Review by rolfedh — COMMENTED** [review]
LGTM

**Pull request merged**